### PR TITLE
[Qualcom] 10371 - CO Detector 

### PR DESCRIPTION
**Pull request by asoloway64**
new pull request template

### CODE REVIEW

File: 10371.xml [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+				<Description><![CDATA[0 = CO NOT detected
+1 = CO Detected]]></Description>
+			</Item>
+			<Item ID="2">
+				<Name>Ambient CO Value</Name>
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration>0-600</RangeEnumeration>
```
Comment (seanmcilroy29) [inline 10371.xml]: RangeEnumeration should be 0..600

All RangeEnumerations should be `..` not `-`

File: 10371.xml [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+				<Description><![CDATA[CO value (in ppm) received from web for the area.]]></Description>
+			</Item>
+			<Item ID="3">
+				<Name>CO Value</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Mandatory</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration>0-600</RangeEnumeration>
+				<Units>To Be Defined</Units>
```
Comment (seanmcilroy29) [inline 10371.xml]: `To be Defined` can't be a `<units>` value. 
Does the proposed unit need to be registered with SenML?

**Comment by seanmcilroy29** [comment]
The submitted Object is failing validation;

Description | ObjectID and/or     ResourceID | Resolve By
-- | -- | --
Incorrect   Unit value (To Be Defined) | 10371:2 | Submitter
Incorrect   Range value (0-600) | 10371:2 | Submitter
Incorrect   Unit value (To Be Defined) | 10371:3 | Submitter
Incorrect   Range value (0-600) | 10371:3 | Submitter
Incorrect   Unit value (To Be Defined) | 10371:4 | Submitter
Incorrect   Range value (0-600) | 10371:4 | Submitter
Incorrect   Unit value (To Be Defined) | 10371:5 | Submitter
Incorrect   Range value (0-600) | 10371:5 | Submitter
Incorrect   Unit value (To Be Defined) | 10371:6 | Submitter
Incorrect   Range value (0-600) | 10371:6 | Submitter
Incorrect   Unit value (To Be Defined) | 10371:7 | Submitter
Incorrect   Range value (0-600) | 10371:7 | Submitter
Incorrect   Unit value (To Be Defined) | 10371:8 | Submitter
Incorrect   Range value (0-600) | 10371:8 | Submitter
Incorrect   Unit value (To Be Defined) | 10371:9 | Submitter
Incorrect   Range value (0-600) | 10371:9 | Submitter

**Comment by asoloway64** [comment]
Fixed all of the ranges and units.

**Comment by asoloway64** [comment]
Changed all ranges below to 0..600 and units to “ppm” (per SenML definition).

File: 10371.xml [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+POSSIBILITY OF SUCH DAMAGE.
+
+-->
+
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.openmobilealliance.org/tech/profiles/LWM2M-v1_1.xsd">
+	<Object ObjectType="MODefinition">
+		<Name>CO Detector</Name>
+		<Description1><![CDATA[This is an alarm that should be raised if the sensor detects higher CO level.]]></Description1>
+		<ObjectID>10371</ObjectID>
+		<ObjectURN>urn:oma:lwm2m:ext:10371</ObjectURN>
```
Comment (seanmcilroy29) [inline 10371.xml]: urn should be `urn:oma:lwm2m:x:10371`

**Comment by mkgillmore** [comment]
Group agrees to close pull request and create a new one in the IPSO repository. 7/2/2020

**Pull request closed without merging**